demo/dsa-accel-config-demo/idxd-reset.patch

```diff
@@ -0,0 +1,26 @@
+From 07e1137a71021dc02df98b942a260b70ff852ca5 Mon Sep 17 00:00:00 2001
+From: Oleg Zhurakivskyy <oleg.zhurakivskyy@intel.com>
+Date: Tue, 3 May 2022 15:28:05 +0300
+Subject: [PATCH] idxd: Ignore reset device error
```

Comment:
it's not clear why this is needed. have we reported this bug to `idxd-config` upstream?